docs/ConfigureScalarDL.md

```diff
@@ -215,6 +221,105 @@ auditor:
 
 To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
 
+### AWS RDS Databases(MySQL PostgreSQL, Oracle, SQL Server, Aurora), Azure Databases for MySQL, PostgreSQL
+
+#### Requirements
+
+* The JDBC connection URL (`JDBC_CONNECTION_URL`) format may vary according to the cloud service provider and database you choose. Please check the official documentation and confirm the connection URL format.
+* For `Azure Database for MySQL` use <ACCOUNT_USERNAME@HOST_NAME> instead of <ACCOUNT_USERNAME> for `username` and `dbUsername` values when updating the configuration files.
```

Comment:
Ditto

Reply:
Removed the requirements section for now.